src/onelogin/saml2/errors.py

```diff
@@ -50,3 +52,73 @@ def __init__(self, message, code=0, errors=None):
 
         Exception.__init__(self, message)
         self.code = code
+
+
+class OneLogin_Saml2_ValidationError(Exception):
+    """
+    This class implements another custom Exception handler, related
+    to exceptions that happens during validation process.
+    Defines custom error codes .
+    """
+
+    # Validation Errors
+    UNSUPPORTED_SAML_VERSION = 0
```

Comment:
I personally don't like the use of "codes" in exceptions, since users of the library need to understand they need to inspect a specific property in the exception, but I do see how this is keeping consistency with the implementation of the existing `OneLogin_Saml2_Error`. 

A better solution would be to have an exception for each of these individually, that possibly inherit from a base class exception class `OneLogin_Saml2_ValidationError`.